charts/prometheus-elasticsearch-exporter/ci/security-context.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-# Set default security context for kubernetes
+# Unset pod level securityContext
 
-securityContext:
-  disable: true
+podSecurityContext: null
```

Comment:
```suggestion
podSecurityContext: {}
```

Reply:
This does not produce the desired outcome (securityContext to be removed from the pod level), at least with Helm v3.10.2 on my laptop. Helm “merges“ the empty dict with the dict from `values.yaml` and the end result is thus the dict from values.yaml. The idea here is to completely remove the securityContext and this only works by setting it to "null" causing Helm to unset the value of "podSecurityContext" instead of merging it.